### PR TITLE
frugal: init at 3.14.2

### PR DESCRIPTION
**Pull request by diogox**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Missing a top level entry

**Comment by diogox** [comment]
My bad, it's been added :)

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2457,6 +2457,12 @@
     githubId = 40633781;
     name = "Sergei S.";
   };
+  diogox = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:2460]: Please put this into a separate commit with the message ``maintainers: add diogox`` and sort it alphabetically.

Comment (diogox) [reply maintainers/maintainer-list.nix]: Done :raised_hands:

**Comment by lukegb** [comment]
3.14.3 is now out, but I'm going to merge this to give you a head start. Welcome!
